src/pymor/core/config.py

```diff
@@ -2,9 +2,18 @@
 # Copyright 2013-2021 pyMOR developers and contributors. All rights reserved.
 # License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
 
+# workaround for dolfin+dune incompat https://github.com/pymor/pymor/issues/1397
```

Comment:
If this needs to be in `config.py` please move it into `_get_fenics_version` if possible

Reply:
also thought about that, now done